charts/airbyte/README.md

```diff
@@ -29,7 +29,7 @@
 | `webapp.replicaCount`        | Number of webapp replicas                                        | `1`              |
 | `webapp.image.repository`    | The repository to use for the airbyte webapp image.              | `airbyte/webapp` |
 | `webapp.image.pullPolicy`    | the pull policy to use for the airbyte webapp image              | `IfNotPresent`   |
-| `webapp.image.tag`           | The airbyte webapp image tag. Defaults to the chart's AppVersion | `0.29.21-alpha`  |
+| `webapp.image.tag`           | The airbyte webapp image tag. Defaults to the chart's AppVersion | `0.29.22-alpha`  |
```

Comment:
this value is bumped automatically or need a manual update?

Reply:
It could probably be auto bumped by `bumpversion`. I need to take a closer look at what airflow is doing for their helm-chart releases. They do a separate release for changes to the helm-chart. ex: https://github.com/apache/airflow/releases/tag/helm-chart%2F1.2.0

If that is okay with the Airbyte team, I can look at going that direction.